src/ui.rs

```diff
@@ -171,6 +172,18 @@ impl Ui {
         Ok(())
     }
 
+    pub fn write_clap_err_and_exit(&mut self, err: clap::Error) -> ! {
+        // Color parsing must never panic, otherwise we'll never print help.
+        let color = use_color(color_setting(&self.settings));
+        let clap_str = if color {
+            err.render().ansi().to_string()
+        } else {
+            err.render().to_string()
+        };
+        self.write(clap_str.as_str()).ok();
```

Comment:
Should be `write_err()`, right? I think `clap` would have printed it to stderr.

Reply:
Yes, it does go to stderr. Did you mean `write_error()`? That uses a formatter, although it seems to give the right output.

Reply:
Yes, I meant `write_error()`. Good point about the formatter. `stderr_formatter().write_str()` is probably better so we don't want to add color codes on top of what clap adds.